hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseCryptoHandler.java

```diff
@@ -56,4 +57,9 @@ protected long getStakedId(
     public static AccountID asAccount(final long num) {
         return AccountID.newBuilder().accountNum(num).build();
     }
+
+    @NonNull
+    public static TokenID asToken(final long num) {
```

Comment:
Nit: This should probably take shard and realm as well.

Reply:
This uses default shard and realm 0.

Reply:
And we need to stop using "default" for those.  The method should take the shard and realm, and if the caller uses default, so be it, but we don't want to _continue_ locking the whole system to that default.